packages/api-nest/src/queue/processors/content-processor.service.ts

```
@@ -226,11 +268,333 @@ export class ContentProcessorService
+      return {
+        success: false,
+        error: errorMessage,
+      }
+    }
+  }
+
+  /**
+   * Convert PDF plain text to HTML format for consistent storage
+   */
+  private convertPdfTextToHtml(text: string, pageCount?: number): string {
+    if (!text) return ''
```

ESLint

[error] Expected { after 'if' condition.

(curly)